### PR TITLE
dart: add French translation

### PR DESCRIPTION
**Pull request by antoine-amara**
* translate the dart command from english to french

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.fr/common/dart.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+> Ligne de commande pour gérer un projet Dart.
+> Plus d'informations : <https://dart.dev/tools/dart-tool>.
+
+- Initialise un nouveau projet Dart dans un dossier du même nom:
+
+`dart create {{project_name}}`
+
+- Exécuter un fichier Dart:
+
+`dart run {{path/to/file.dart}}`
```
Comment (kbdharun) [inline pages.fr/common/dart.md:12]:
```suggestion
`dart run {{chemin/vers/fichier.dart}}`
```
[Reference](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/translation-templates/common-arguments.md)

File: pages.fr/common/dart.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`dart test`
+
+- Mettre à jour les dépendances d'un projet pour supporter null-safety:
+
+`dart pun upgrade --null-safety`
+
+- Compiler un fichier Dart vers un binaire natif:
+
+`dart compile exe {{path/to/file.dart}}`
```
Comment (kbdharun) [inline pages.fr/common/dart.md:28]:
```suggestion
`dart compile exe {{chemin/vers/fichier.dart}}`
```

File: pages.fr/common/dart.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
+# dart
+
+> Ligne de commande pour gérer un projet Dart.
+> Plus d'informations : <https://dart.dev/tools/dart-tool>.
+
+- Initialise un nouveau projet Dart dans un dossier du même nom:
+
+`dart create {{project_name}}`
```
Comment (kbdharun) [inline pages.fr/common/dart.md:8]: Translate `{{project_name}}` too

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have some suggestions for this page.

**Comment by antoine-amara** [comment]
Thank you for the review @kbdharun , I pushed the updates

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

File: pages.fr/common/dart.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+- Exécuter les tests unitaire pour le projet courant:
+
+`dart test`
+
+- Mettre à jour les dépendances d'un projet pour supporter null-safety:
+
+`dart pun upgrade --null-safety`
+
+- Compiler un fichier Dart vers un binaire natif:
```
Comment (navarroaxel) [inline pages.fr/common/dart.md:26]:
```suggestion
- Compiler un fichier Dart vers un binaire natif :
```

File: pages.fr/common/dart.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+- Télécharger les dépendences pour le projet courant:
+
+`dart pub get`
+
+- Exécuter les tests unitaire pour le projet courant:
+
+`dart test`
+
+- Mettre à jour les dépendances d'un projet pour supporter null-safety:
```
Comment (navarroaxel) [inline pages.fr/common/dart.md:22]:
```suggestion
- Mettre à jour les dépendances d'un projet pour supporter null-safety :
```

File: pages.fr/common/dart.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+
+- Exécuter un fichier Dart:
+
+`dart run {{chemin/vers/fichier.dart}}`
+
+- Télécharger les dépendences pour le projet courant:
+
+`dart pub get`
+
+- Exécuter les tests unitaire pour le projet courant:
```
Comment (navarroaxel) [inline pages.fr/common/dart.md:18]:
```suggestion
- Exécuter les tests unitaire pour le projet courant :
```

File: pages.fr/common/dart.md [hunk 7]
```diff
@@ -0,0 +1,28 @@
...
+
+- Initialise un nouveau projet Dart dans un dossier du même nom:
+
+`dart create {{nom_du_projet}}`
+
+- Exécuter un fichier Dart:
+
+`dart run {{chemin/vers/fichier.dart}}`
+
+- Télécharger les dépendences pour le projet courant:
```
Comment (navarroaxel) [inline pages.fr/common/dart.md:14]:
```suggestion
- Télécharger les dépendences pour le projet courant :
```

File: pages.fr/common/dart.md [hunk 8]
```diff
@@ -0,0 +1,28 @@
+# dart
+
+> Ligne de commande pour gérer un projet Dart.
+> Plus d'informations : <https://dart.dev/tools/dart-tool>.
+
+- Initialise un nouveau projet Dart dans un dossier du même nom:
+
+`dart create {{nom_du_projet}}`
+
+- Exécuter un fichier Dart:
```
Comment (navarroaxel) [inline pages.fr/common/dart.md:10]:
```suggestion
- Exécuter un fichier Dart :
```

File: pages.fr/common/dart.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
+# dart
+
+> Ligne de commande pour gérer un projet Dart.
+> Plus d'informations : <https://dart.dev/tools/dart-tool>.
+
+- Initialise un nouveau projet Dart dans un dossier du même nom:
```
Comment (navarroaxel) [inline pages.fr/common/dart.md:6]:
```suggestion
- Initialise un nouveau projet Dart dans un dossier du même nom :
```

**Review by navarroaxel — APPROVED** [review]

**Pull request merged**